cmd/verify/quota/cmd.go

```diff
@@ -67,7 +67,12 @@ func run(cmd *cobra.Command, _ []string) (err error) {
 		if strings.Contains(fmt.Sprintf("%s", err), "STS") {
 			r.OCMClient.LogEvent("ROSAInitCredentialsSTS", nil)
 		}
-		r.Reporter.Errorf("Error creating AWS client: %v", err)
+		if strings.Contains(fmt.Sprintf("%s", err), "InvalidClientTokenId") {
+			r.Reporter.Errorf(
+				"Error creating AWS client: Invalid AWS Credentials. Update credentials in ~/.aws/credentials")
```

Comment:
@willkutler Does it make more sense to link to the ROSA docs on how to configure your AWS client? https://docs.openshift.com/rosa/rosa_install_access_delete_clusters/rosa_getting_started_iam/rosa-config-aws-account.html#rosa-configuring-aws-account_rosa-config-aws-account

`~/.aws/credentials` may not be how the customer is configuring their AWS credentials e.g they have set the environment variables `AWS_ACCESS_KEY_ID` and `AWS_SECRET_ACCESS_KEY`...